wordSearch.java

```diff
@@ -1,2 +1,17 @@
-package PACKAGE_NAME;public class wordSearch {
+import java.util.Scanner;
+
+public class wordSearch {
+
+    public static void main(String[] args) {
```

Comment:
We discussed about moving this content into a function/method and then instantiating the object to call the method - remember? Please update.